src/ui/public/kfetch/kfetch.test.ts

```diff
@@ -46,21 +47,21 @@ describe('kfetch', () => {
   it('should use supplied request method', async () => {
     fetchMock.post('*', {});
     await kfetch({ pathname: 'my/path', method: 'POST' });
-    expect(fetchMock.lastOptions('*').method).toBe('POST');
+    expect(fetchMock.lastOptions()!.method).toBe('POST');
   });
 
   it('should use supplied Content-Type', async () => {
     fetchMock.get('*', {});
     await kfetch({ pathname: 'my/path', headers: { 'Content-Type': 'CustomContentType' } });
-    expect(fetchMock.lastOptions('*').headers).toMatchObject({
+    expect(fetchMock.lastOptions()!.headers).toMatchObject({
```

Comment:
what's ()! do?

Reply:
`fetchMock.lastOptions()` returns an optional value, so Typescript requires a check for undefined. `!` is an operator to tell the compiler that the value is not optional. 
I would not do this in non-test code since lying to the compiler can cause runtime bugs but the worst that can happen here is that the tests don't pass.

The Typescript wiki explains it more eloquently https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator